student/propose.php

```diff
@@ -91,6 +91,8 @@
 </html>
 
 <?php
+
+
```

Comment:
@uday-kokitkar _

Reply:
@Sanjivn I have fixed the issue. Please re-review.